spyder/plugins/application/container.py

```diff
@@ -297,21 +311,36 @@ def _check_updates_ready(self):
                         "<code>conda install spyder={}</code><br><br>"
                     ).format(latest_release)
                 else:
-                    content = _(
-                        "Click <a href=\"{}\">this link</a> to "
-                        "download it.<br><br>"
-                    ).format(url_r)
+                    if os.name == 'nt' and is_pynsist():
+                        box.setStandardButtons(QMessageBox.Yes |
+                                               QMessageBox.No)
+                        content = _(
+                            "You want to download and install the latest "
+                            "version of Spyder?<br><br>"
```

Comment:
```suggestion
                            "Would you like to automatically download and install"
                            "the latest version of Spyder?<br><br>"
```

Fix bad grammar and clarify that it will be done automatically for you